pkg/spacebinding/spacebindinglister_test.go

```diff
@@ -235,4 +235,199 @@ func TestNewSpaceBindingLister(t *testing.T) {
 			})
 		}
 	})
+
+	t.Run("recursive list for space with disable Inheritance", func(t *testing.T) {
+		// given
+		spaceAMur := NewMasterUserRecord(t, "space-a-user", TargetCluster(test.MemberClusterName), TierName("deactivate90"))
+		spaceBMur := NewMasterUserRecord(t, "space-b-user", TargetCluster(test.MemberClusterName), TierName("deactivate90"))
+		spaceCMur := NewMasterUserRecord(t, "space-c-user", TargetCluster(test.MemberClusterName), TierName("deactivate90"))
+		spaceDMur := NewMasterUserRecord(t, "space-d-user", TargetCluster(test.MemberClusterName), TierName("deactivate90"))
+		spaceFMur := NewMasterUserRecord(t, "space-f-user", TargetCluster(test.MemberClusterName), TierName("deactivate90"))
+		// we have a parent space
+		spaceA := spacetest.NewSpace(test.HostOperatorNs, "Space-A",
+			spacetest.WithTierName("advanced"),
+			spacetest.WithSpecTargetCluster(test.MemberClusterName),
+			spacetest.WithDisableInheritance(false),
```

Comment:
Thanks for looking into it.

Unless I'm missing something, I just tried removing all the `spacetest.WithDisableInheritance(false)` and all tests are passing. It is set to false by default, as you wrote here https://github.com/codeready-toolchain/api/blob/6074997e1bd7b81cc1d21617c69f9a6a897e615a/api/v1alpha1/space_types.go#L76.

Not a big issue, so feel free to leave it as is if you think it's more clear.

Reply:
Francisc is right, leaving it unset means `false`, which is exactly what we need - disabled by default. It would be different if you would define the field as a pointer, in that case leaving it unset would mean `nil`, but this is not this case.
I didn't see this line as confusing - could be used to document what the test is focused on. 
That being said, either way is fine for me :peace_symbol: